docs/plugins/integrations/snmp.asciidoc

```diff
@@ -18,4 +18,127 @@ END - GENERATED VARIABLES, DO NOT EDIT!
 
 === SNMP Integration Plugin
 
-COMING SOON!
+* Plugin version: v4.0.0
+* Released on: 2024-05-03
+* https://github.com/logstash-plugins/logstash-integration-snmp/blob/main/CHANGELOG.md[Changelog]
+
+experimental[]
+
+.Technical Preview
+****
+The new `integration-snmp` plugin, and its component plugins--`input-snmp` and `input-snmptrap`--are available in _Technical Preview_ at {ls} 8.14.0.
+
+Pre-4.0.0 versions of the `input-snmp` plugin are bundled with {ls} by default, and will not be upgraded to the snmp integration without your deliberate action. 
+(If you do want to opt into the Technical Preview for the `integration-snmp` plugin, run `bin/logstash-plugin install logstash-integration-snmp`.)
+
+Be aware of behavioral and mapping differences between current stand-alone plugins and the new versions included in the `integration-snmp`. 
+The information in this topic can help. 
+****
+
+==== Description
+
+The SNMP integration plugin includes:
+
+* SNMP input plugin
+* SNMPtrap input plugin
+
+// Uncomment and replace with above bullets after generated the initial versions of the target files
+// - {logstash-ref}/plugins-input-snmp.html[SNMP input plugin]
+// - {logstash-ref}/plugins-input-snmptrap.html[Snmptrap input plugin]
+
+The new the `logstash-integration-snmp` plugin combines the 
+`logstash-input-snmp` and `logstash-input-snmptrap` plugins into one integrated plugin that encompasses
+the capabilities of both. 
+This integrated plugin package provides better alignment in snmp processing, better resource managenment, 
+easier package maintenance, and a smaller installation footprint. 
+
+
+[id="plugins-{type}s-{plugin}-migration"]
+==== Migrating to `logstash-integration-snmp` from individual plugins
+
+You'll retain and expand the functionality of existing stand-alone plugins, but in a more compact, integrated package. 
+In this section, we'll note mapping and behavioral changes, and explain how to preserve current behavior if needed.
+
+[IMPORTANT] 
+--
+Uninstall the `logstash-input-snmp` and `logstash-input-snmptrap` plugins _before_ you install the `logstash-integration-snmp` plugin. 
+
+* `bin/logstash-plugin remove logstash-input-snmp` +
+* `bin/logstash-plugin remove logstash-input-snmptrap`
+--
```

Comment:
```suggestion
```
This shouldn't be necessary as installing the snmp integration plugin uninstalls the individual plugins it provides.